speech/api/Recognize/Recognize.cs

```diff
@@ -285,6 +290,76 @@ static object SyncRecognizeMultipleChannels(string filePath, int channelCount)
         }
         // [END speech_transcribe_multichannel_beta]
 
+        // [START speech_transcribe_diarization]
+        static object SyncRecognizeMultipleSpeakers(string filePath, int numberOfSpeakers)
+        {
+            var speech = SpeechClient.Create();
+
+            // Create the transcription request
```

Comment:
nit: Consider removing this comment, since you are in one go creating the request, executing the call and storing the result in response.

Reply:
Done.